src/lib/_navigation.scss

```diff
@@ -1,4 +1,4 @@
-$navigationBtnWidth: 50px;
+$navigationBtnWidth: 40px;
```

Comment:
Why reduce the button width? What if you left everything the same, but changed the left and right of .bp-navigate-left and .bp-navigate-right, would that work?

Reply:
Yeah, you're right I could keep the button width the same, I guess I was just trying to keep the button about in the same place as it ever was (though what is 5px vs 10px really)